.gitignore

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
great idea to add a .gitignore. Maybe we can change the pattern so that all __pycache__ and all xml files inside .idea are ignored?